dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
@bbeck -
I meant preserve the existing ECS parent pom (that builds docker) as is, and eliminate the middle layer of the dropwizard parent pom, in favor of importing the dropwizard bom in each project as you show in your example. I think we could go either way, and having the middle layer there gives us some additional flexibility with adding other things to the build specifically for Dropwizard.